semantic error: Type mismatch while appending to a list, found ('str' and 'i32').
 --> tests/errors/test_append_type_mismatch.py:6:5
  |
6 |     l.append('a')
  |     ^^^^^^^^^^^^^ 
